tests/test_yum.rb

```diff
@@ -26,34 +27,20 @@ class TestYum < CiscoTestCase
   @skip_unless_supported = 'yum'
 
   def select_pkg
+    path = File.expand_path('../yum_package.yaml', __FILE__)
+    skip('Cannot find tests/yum_package.yaml') unless File.file?(path)
+    pkginfo = YAML.load(File.read(path))
+
     # rubocop:disable Style/ClassVars
-    @@pv = Platform.image_version
-    case @@pv
-    when /7.0\(3\)I2\(1\)/
-      info 'Testing Patch For Camden Release Image'
-      @@pkg_filename = 'n9000_sample-1.0.0-7.0.3.x86_64.rpm'
-      @@pkg = 'n9000_sample'
-      @@pkg_ver = '1.0.0-7.0.3'
-    when /7.0\(3\)I3\(1\)/
-      info 'Testing Patch For Dublin Release Image'
-      @@pkg_filename = 'CSCuxdublin-1.0.0-7.0.3.I3.1.lib32_n9000.rpm'
-      @@pkg = 'CSCuxdublin'
-      @@pkg_ver = '1.0.0-7.0.3.I3.1'
-    when /7.0\(4\)I4\(1\)/
-      # TBD: Add Dublin Plus RPM when it becomes available.
-      # info 'Testing Patch For Dublin Plus Release Image'
-    when /7.0\(3\)F1\(0.313\)/
-      # TBD: Update with actual fretta_camden release patch when ready
-      info 'Testing Patch For Fretta Camden Release Image'
-      @@pkg_filename = 'nxos.sample-n8k_EOR-1.0.0-7.0.3.F1.1.lib32_nxos.rpm'
-      @@pkg = 'nxos.sample-n8k_EOR'
-      @@pkg_ver = '1.0.0-7.0.3.F1.1'
-    else
-      skip "Available patches are not compatible with this image: #{@@pv}"
-    end
```

Comment:
It would be helpful to include a before & after example of the text you're gsub'ing